### DISCUSSION TITLE
List of Notification-Toast

### DISCUSSION THREAD

**Discussion by Lino-telekom** [telekom/scale#1162] [Q&A]
I want to show on a website all elements, which are stored inside an array in different notification-toasts. For this I used a for-loop. But experimenting with listing the elements I always get the same result. The notifications are stacked above each other. 
Is there a solution where are many notification-toasts listed below each other? 

In addition, I would like to know how I can delete the entry in the array when the dismiss is clicked.

Regards Lino

**Comment by acstll** [comment]
> Is there a solution where are many notification-toasts listed below each other?

currently not, but this is something we want to provide in the near future as we improve the component to get it out of `beta`. 

Best.

**Reply by acstll** [reply]
@Lino-telekom there's a new version of the Notification component in which is easy to stack toasts… it's not ready yet, but should be soon. Here's a the on-going work: https://github.com/telekom/scale/pull/1180
